### PR TITLE
chore: Remove reference to paver update_db.

### PR DESCRIPTION
**Pull request by dianakhuang**
We are planning on deprecating this method,
and this piece of code has been commented out
for a while anyway.

### CODE REVIEW

**Review by davidjoy — APPROVED** [review]

File: provision-lms.sh [hunk 1]
```diff
@@ -26,7 +26,6 @@ for app in "${apps[@]}"; do
 done
 
 # Run edxapp migrations first since they are needed for the service users and OAuth clients
```
Comment (robrap) [inline provision-lms.sh:28]: @dianakhuang: Why did you leave in the comment? Also, did you do any research about why this was here in the first place and why it was commented out?

**Review by robrap — CHANGES_REQUESTED** [review]

File: provision-lms.sh (hunk 1 above)
Comment (dianakhuang) [reply provision-lms.sh:28]: I left in the comment because it's still running migrations. We're just not using `paver update_db` to run them.

Comment (robrap) [reply provision-lms.sh:28]: Oops. I didn't realize those were migration commands.
1. Nit: Do you mind removing the newline between? That's just personal preference, so ok if you leave it.
2. Is this another place we could switch to the edx-platform Makefile migrate calls, and again there is the `showmigrations` question from https://github.com/openedx/edx-platform/pull/32705#issuecomment-1632843276.

**Review by robrap — APPROVED** [review]

File: provision-lms.sh (hunk 1 above)
Comment (dianakhuang) [reply provision-lms.sh:28]: Yeah, I have the new migrate commands run `showmigrations` as well, so I think replacing these with `make migrate` will be effectively the same.

**Pull request merged**